apps/web/scripts/kernel/messaging.ts

```diff
@@ -38,23 +39,37 @@ export const initializeMessaging = async (): Promise<void> => {
 		const callbackRegistry: Record<string, MessengerCallback> = {};
 
 		runtime.onConnect.addListener((port) => {
-			const handleInComingMessage = async (message: EncryptedMessage) => {
+			console.log('Connecting with:', port.sender?.url);
```

Comment:
remove this log

Reply:
Might we put this log here? Each time debug with sdk or service worker, I always need to know what happens. So if it has description, I want to put it here for later

Reply:
we may need an util that logs but conditional, e.g only on development - would be okay if we leave it this way for now.